pkg/cable/vxlan/vxlan.go

```diff
@@ -257,7 +257,8 @@ func (v *vxlan) getVxlanVtepIPAddress(ipAddr string) (net.IP, error) {
 	return vxlanIP, nil
 }
 
-func (v *vxlan) ConnectToEndpoint(endpointInfo *natdiscovery.NATEndpointInfo) (string, error) {
+func (v *vxlan) ConnectToEndpoint(endpointInfo *natdiscovery.NATEndpointInfo) (string, error) { // nolint // Supposed to be complicated
```

Comment:
I should move all of these cabledrive changes to an earlier commit, to better segment from globalnet changes.